uri-schemes.md

```diff
@@ -124,6 +126,73 @@ HTTP Status code: NOT 2XX, idealy the 4XX if it's user error and 5XX if it's the
 
 json object can have other properties too, but currently they are ignored by core.
 
+## **DCLOGIN** <a name="DCLOGIN"></a>
+
+|                          |                       |
+| ------------------------ | --------------------- |
+| Scheme                   | `DCLOGIN:`            |
+| Used for                 | Account setup         |
+| Related Terms\*          | Account Login QR code |
+| Available on             | draft                 |
+| Decoded by the core \*\* | draft                 |
+| Other apps using it      | none, only DeltaChat  |
+
+### Syntax
+
+```
+DCLOGIN:CredentialsJSON#minVersion
```

Comment:
what's the reason to put the version at the end?  Seems simpler to parse chunks from the front scanning for the delimiter (also note my earlier comment on keeping the same delimiter)

Reply:
When we want a new format with the same schema, so deltachat can detect that the schema is too new and prompt the user to update, basically future proofing